### PR TITLE
feat(cal): index.js

### PR DESCRIPTION
**Pull request by brain-st0rm**
create calculator in index.js and perform test also

### CODE REVIEW

**Review by SubhajitBar — APPROVED** [review]

File: app/index.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+const mul = function (a,b){
+    return a*b
+}
+const div = function (a,b){
+    return a/b
+}
 
-module.exports = add;
\ No newline at end of file
+module.exports = {add, sub, mul, div};
```
Comment (echo-ech0) [inline app/index.js:14]: Re-check module.exports

File: test/index.test.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
-const add = require("../app/index");
+const cal = require("../app/index");
 
 test('Adds 1 + 2 to equal 3', ()=>{
-    expect(add(1,2)).toBe(3);
+    expect(cal.add(1,2)).toBe(3);
 });
 
-test('Adds 5 + 9 to equal 14', ()=>{
-    expect(add(5,9)).toBe(14);
```
Comment (echo-ech0) [inline test/index.test.js:8]: explain code removal

Comment (brain-st0rm) [reply test/index.test.js:8]: already have a similar test case above

File: app/index.js (hunk 1 above)
Comment (brain-st0rm) [reply app/index.js:14]: Re-check Done. It's working.

**Review by echo-ech0 — APPROVED** [review]

**Pull request merged**